data/mil2525c_b2/test_data/Complete_test_scripts/scripts/FilterCSVMilSymbols.py

```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------------
+# Copyright 2015 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+#import CSV Module
+import csv
+#import arcpy
+import arcpy
+
+#filtergeom = 'POINT'
+#filterappendix = 'A'
+inputFileName = arcpy.GetParameterAsText(0)
+outputfolder = arcpy.GetParameterAsText(1)
+
+outputFileDict = {"AppendixA_Points.csv": ["A", "POINT"], \
+                  "AppendixB_Points.csv": ["B", "POINT"], \
+                  "AppendixB_lines.csv": ["B", "LINE"], \
+                  "AppendixB_areas.csv": ["B", "AREA"], \
+                  "AppendixC_Points.csv": ["C", "POINT"], \
+                  "AppendixC_lines.csv": ["C", "LINE"], \
+                  "AppendixC_areas.csv": ["C", "AREA"], \
+                  "AppendixD_Points.csv": ["D", "POINT"], \
+                  "AppendixE_Points.csv": ["E", "POINT"], \
+                  "AppendixG_Points.csv": ["G", "POINT"]}
+
+for key in outputFileDict:
+    dictionaryEntry = outputFileDict[key]
+    filterappendix = dictionaryEntry[0]
+    filtergeom = dictionaryEntry[1]
+    print(key, outputFileDict[key], filterappendix, filtergeom)
+
+    if outputfolder == "" or outputfolder is None:
+        outputfolder = "C:\\Users\\dani8200\\Documents\\MilitarySymbology\\Test Datasets\\PythonforTestDatasets\\Cresults"
```

Comment:
Generally best practice not to point to your user name/path for default location.
If the whole repo has been downloaded, the data should be in a known location relative to the script.
Here is an example (thanks @csmoore ) of how to get at it:
    if All_ID_Mapping_Original_csv == '':
        currentPath = os.path.dirname(__file__)
        All_ID_Mapping_Original_csv = os.path.normpath(os.path.join(currentPath, "../../../../../../data/mil2525c_b2/core_data/gdb/schema_source_data/domains_input/All_ID_Mapping_Original.csv"))
        print str(All_ID_Mapping_Original_csv)